connector/connect/client/jvm-internal/src/test/scala/org/apache/spark/sql/connect/client/util/DummySparkConnectService.scala

```diff
@@ -0,0 +1,194 @@
+/*
```

Comment:
This got moved from SparkConnectClientSuite.
`withNettyDummySparkConnectService` helper was added.